modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -149,6 +149,14 @@ TEST_P(Test_ONNX_layers, Clip)
     testONNXModels("clip", npy);
 }
 
+TEST_P(Test_ONNX_layers, ReduceMean)
+{
+    if (target != DNN_TARGET_CPU)
+        throw SkipTestException("Only CPU is supported");
```

Comment:
Can you please check if nonn-CPU cases still do not work?

Reply:
reduce_mean3d (avepool3d) works only on CPU

Reply:
I see. Can we split test in two?